src/pages/add.astro

```diff
@@ -97,6 +97,7 @@ import '../styles/styles.css';
       url.searchParams.set('digits', digitsVal);
       url.searchParams.set('numbers', numbersVal);
       url.searchParams.set('speed', speedVal);
+      url.searchParams.set('arithmeticType', '0');
```

Comment:
Is there any way we can make a TypeScript `enum` for this and parse it to/from the enum? Rather than using a magic value.